static/src/javascripts/projects/common/modules/commercial/membership-messages.js

```diff
@@ -78,12 +88,20 @@ define([
 
     function show(edition, message) {
         var data = defaults({ linkHref: formatEndpointUrl(edition, message) }, message.data, defaultData);
+        // Rotate through six different colours on successive page views
+        var colour = storage.local.get('gu.alreadyVisited') % 6;
```

Comment:
if you're worried about hidden behaviour you can always make the is-defined check explicit ;)